<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iNI</class>
 <widget class="QMainWindow" name="iNI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>iNode</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>其他信息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>磁盘块地址</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="rowCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>密码</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>iNode</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget_3">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>文件</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>iNode</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
